### PR TITLE
feat(tag): unstable rtl

### PR DESCRIPTION
**Pull request by timeTravelCYN**

### CODE REVIEW

File: src/tag/src/Tag.tsx [hunk 1]
```diff
@@ -150,6 +158,7 @@ export default defineComponent({
           '--close-color-hover': closeColorHover,
           '--close-color-pressed': closeColorPressed,
           '--close-margin': closeMargin,
+          '--close-rtl-margin': closeRtlMargin,
```
Comment (07akioni) [inline src/tag/src/Tag.tsx:161]: close-margin-rtl

File: src/tag/src/Tag.tsx [hunk 2]
```diff
@@ -195,7 +205,12 @@ export default defineComponent({
         {!this.checkable && this.closable ? (
           <NBaseClose
             clsPrefix={mergedClsPrefix}
-            class={`${mergedClsPrefix}-tag__close`}
+            class={[
+              `${mergedClsPrefix}-tag__close`,
+              {
+                [`${mergedClsPrefix}-tag__close--rtl`]: rtlEnabled
```
Comment (07akioni) [inline src/tag/src/Tag.tsx:211]: use class from outermost dom to apply style

File: src/tag/styles/_common.ts [hunk 3]
```diff
@@ -4,5 +4,6 @@ export default {
   closeSizeLarge: '14px',
   // closeSize
   padding: '0 7px',
-  closeMargin: '0 0 0 3px'
+  closeMargin: '0 0 0 3px',
+  closeRtlMargin: '0 3px 0 0'
```
Comment (07akioni) [inline src/tag/styles/_common.ts:8]: closeMarginRtl

File: src/tag/src/Tag.tsx (hunk 2 above)
Comment (timeTravelCYN) [reply src/tag/src/Tag.tsx]: @07akioni done

**Pull request merged**